index.bs

```diff
@@ -894,6 +912,65 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
   </pre>
 </div>
 
+## Update camera focus distance and {{takePhoto()}} ## {#example4}
+
```

Comment:
Perfect, I can add this to the PR. I don't have a codepen and didn't want to create one and mix it in when you'd already provided a few links under the same username.